compiler/rustc_borrowck/src/diagnostics/region_errors.rs

```diff
@@ -694,4 +698,24 @@ impl<'a, 'tcx> MirBorrowckCtxt<'a, 'tcx> {
             );
         }
     }
+
+    fn suggest_adding_lifetime_params(
+        &self,
+        diag: &mut Diagnostic,
+        sub: RegionVid,
+        sup: RegionVid,
+    ) {
+        if let (Some(sub), Some(sup)) = (self.to_error_region(sub), self.to_error_region(sup)) {
+            let f = |reg| {
```

Comment:
If the function is converted to let else, I don't think having this as a closure helps much.

Reply:
Done